test/dialect/AIEVec/transfer-read-splat.mlir

```diff
@@ -0,0 +1,52 @@
+// RUN: aie-opt %s -canonicalize-for-aievec -split-input-file | FileCheck %s
```

Comment:
Javier, could you please also add a test case for folding mul and add, and a test case for broadcastOp?

Reply:
Sorry, I missed one word "separately."